org.flatpak.EasyEdit.yml

```diff
@@ -0,0 +1,90 @@
+id: org.flatpak.EasyEdit
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: easyedit
+rename-icon: org.flatpak.EasyEdit
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  # Add sandbox overrides
+  - --env=ELECTRON_DISABLE_SANDBOX=1
+  - --filesystem=host
+modules:
+  - name: EasyEdit
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - mkdir -p /app/bin/resources/usr/share/metainfo/
+      - mkdir -p /app/bin/locales/
+      - cp -r * /app/bin/ 
+      - mv -f /app/bin//usr/share/metainfo/easyedit.appdata.xml /app/bin/resources/usr/share/metainfo/
+      - mv -f /app/bin/*.pak /app/bin/locales/
+      - mv -f /app/bin/locales/resources.pak /app/bin/
+      - mv -f /app/bin/app.asar /app/bin/resources/
+      - mv -f /app/bin/app-update.yml /app/bin/resources/
+      - cp -r /app/bin/easyedit.png /app/bin/resources/
+      - chmod 755 /app/bin/easyedit
+      - mkdir -p /app/share/applications
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - mkdir -p /app/share/metainfo
+      - rm -f /app/share/applications/org.flatpak.EasyEdit.desktop
+      - cp /app/bin/easyedit.png /app/share/icons/hicolor/256x256/apps/org.flatpak.EasyEdit.png
+      # Remove sandbox dependency
+      - rm -f /app/bin/chrome-sandbox
+      - |
+        cat > /app/share/metainfo/org.flatpak.EasyEdit.appdata.xml <<'EOF'
+        <?xml version="1.0" encoding="UTF-8"?>
+        <component type="desktop-application">
+            <id>org.flatpak.EasyEdit</id>
+            <name>EasyEdit</name>
+            <summary>Simple text editor</summary>
+            <metadata_license>MIT</metadata_license>
+            <project_license>MIT</project_license>
+            <description>
+                <p>A simple text editor for everyday use</p>
+            </description>
+            <screenshots>
+              <screenshot type="default">
+                <image>https://raw.githubusercontent.com/gcclinux/EasyEdit/refs/heads/main/screenshots/sample011.png</image>
+                <caption>PreviewPanel</caption>
+              </screenshot>
+              <screenshot type="default">
+                <image>https://raw.githubusercontent.com/gcclinux/EasyEdit/refs/heads/main/screenshots/sample010.png</image>
+                <caption>SidebySide</caption>
+                </screenshot>
+            </screenshots>
+            <developer_name>Ricardo Wagemaker</developer_name>
+            <url type="homepage">https://github.com/gcclinux/EasyEdit</url>
+            <releases>
+                <release version="1.3.5" date="2024-03-19"/>
+            </releases>
+        </component>
+        EOF
+      - |
+        cat > /app/share/applications/org.flatpak.EasyEdit.desktop <<'EOF'
```

Comment:
no. do use cat like that.

also there is a .desktop in the archive so why do you need to do that?